packages/web/src/components/upload/UploadChip.tsx

```diff
@@ -17,7 +16,25 @@ const messages = {
   firstArtistPlaylist: 'Upload Your First Playlist'
 }
 
-const UploadChip = ({ type, variant, isArtist = false, isFirst, onClick }) => {
+type UploadChipProps = {
+  type?: 'track' | 'album' | 'playlist'
+  // nav: For display in a nav-like column
+  // card: Looks like a 'Card'
+  // tile: Looks like a 'TrackTile'
```

Comment:
Nit: might want to use JSDoc style comments to make these comments show up with intellisense

```suggestion
  /**
   * nav: For display in a nav-like column
   * card: Looks like a 'Card'
   * tile: Looks like a 'TrackTile'
   */
```

Reply:
but also ik you were just porting the proptypes so nbd